samples/src/main/java/com/example/bigquery/CreateTable.java

```diff
@@ -0,0 +1,46 @@
+package com.example.bigquery;
+
+// [START bigquery_create_table]
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.Field;
+import com.google.cloud.bigquery.LegacySQLTypeName;
+import com.google.cloud.bigquery.Schema;
+import com.google.cloud.bigquery.StandardTableDefinition;
+import com.google.cloud.bigquery.TableDefinition;
+import com.google.cloud.bigquery.TableId;
+import com.google.cloud.bigquery.TableInfo;
+
+public class CreateTable {
+
+  public static void runCreateTable() {
+    // TODO(developer): Replace these variables before running the sample.
+    String datasetName = "my-dataset-name";
+    String tableName = "my_table_name";
+    Schema schema =
+        Schema.of(
+            // LegacySQLTypeName will be updated to StandardSQLTypeName once release rolls out
```

Comment:
Is this a TODO comment? Or a warning to developers? Which release?

This value seems like something that should be as part of the snippet, not something that's required to be provided by the test environment.

Reply:
This is a warning to developers that for now we will be using LegacySQLTypeName since StandardSQLTypeName is not yet supported.

I wanted to move away from using legacy stuff in our sample code so I made a PR to update BQ client libraries: https://github.com/googleapis/java-bigquery/pull/43

This change should go out in release 1.103.0. Due to code freeze (since we're using libraries-bom) the change will be available in maybe 3 weeks.